static/js/controllers/contacts.js

```diff
@@ -141,18 +158,7 @@ var _ = require('underscore'),
           _query();
         } else {
           $scope.filtered = false;
-          liveList = LiveList.contacts;
```

Comment:
This appears to be a hack / work around. Basically if you're "not filtering" it would do a the hacks below (hacks because they are internal implementation details of livelist) and just display a raw livelist. This ended up with the filtering not being honoured correctly in this new situation where we filter all the time, and it made the code more complicated IMO, so I chucked it.